.eslintrc.json

```diff
@@ -34,6 +34,8 @@
       ],
       "rules": {
         "@typescript-eslint/no-use-before-define": 0,
+        "no-shadow": "off",
+        "@typescript-eslint/no-shadow": "error",
```

Comment:
I'd be interested to know why we're doing this.. is it this? https://stackoverflow.com/a/64745652

Reply:
Yeah, exactly that!